.github/workflows/ci.yml

```diff
@@ -31,4 +31,15 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
       - run: bundle exec standardrb
-      - run: COVERAGE=0 TEST_NODE_PARITY=1 bundle exec rake test
+      - name: Run tests with coverage
+        env:
+          COVERAGE: 1
+          TEST_NODE_PARITY: 1
```

Comment:
I think it would make sense to only run with enabled coverage and node parity for Ruby 3.3, to save some worker time.